site/_docs/reference.md

```diff
@@ -1890,7 +1890,7 @@ on a timestamp column. Windows assigned could have overlapping so hopping someti
 
 | Operator syntax      | Description
 |:-------------------- |:-----------
-| HOP(table, DESCRIPTOR(datetime), slide, size) | Indicates a hopping window for *datetime*, covering rows within the interval of *size*, shifting every *slide*.
+| HOP(table, DESCRIPTOR(datetime), slide, size [, time ]) | Indicates a hopping window for *datetime*, covering rows within the interval of *size*, shifting every *slide* and optionally aligned at time.
 
```

Comment:
I prefer to name the time to offset, which is more straight-forward.

Reply:
Agreed. Done!